### PR TITLE
Add remote attestation roles

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

File: control/control.MicroOS.xml [hunk 1]
```diff
@@ -553,6 +573,24 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
...
+	<label>• Same installation that MicroOS role
+• Remote attestation agent based on Keylime and TPM2.0 (required hardware)
+• Configure /etc/keylime.conf and start keylime_agent.service</label>
+        </micro_os_role_ra_agent_description>
+        <micro_os_role_ra_verifier>
+          <!-- TRANSLATORS: a label for a system role -->
+          <label>MicroOS with Remote Attestation (Verifier) [ALPHA]</label>
+        </micro_os_role_ra_verifier>
+        <micro_os_role_ra_verifier_description>
+	<label>• Same installation that MicroOS role
```
Comment (jreidinger) [inline control/control.MicroOS.xml]: I am not sure if it is proper English usage of that "same that" here https://guinlist.wordpress.com/2014/09/08/87-same-as-versus-same-that/
Here I think it should be "Same installation as MicroOS role"

File: control/control.MicroOS.xml [hunk 2]
```diff
@@ -553,6 +573,24 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
...
+• Remote attestation verifier based on Keylime and TPM2.0 (required hardware)
+• Configure /etc/keylime.conf and start keylime_{verifier,registrar}.service</label>
```
Comment (jreidinger) [inline control/control.MicroOS.xml]: I am not sure if here it should not be "Configures" as it is about role.

File: control/control.MicroOS.xml (hunk 1 above)
Comment (aplanas) [reply control/control.MicroOS.xml]: Yes. Done.

File: control/control.MicroOS.xml (hunk 2 above)
Comment (aplanas) [reply control/control.MicroOS.xml]: Should be "configure", in the sense "You configure". I changed it for "edit". Done

**Review by jreidinger — APPROVED** [review]

**Comment by aplanas** [comment]
ping? I'd like to submit this change to Factory. The patterns and the main components are already there : )

**Comment by jreidinger** [comment]
@aplanas you do not have permissions? If so, I can merge it for you

**Comment by aplanas** [comment]
No, I do not have permissions to merge. This can be related: `First-time contributors need a maintainer to approve running workflows`

**Comment by jreidinger** [comment]
@aplanas OK, will do, just let ci pass.

**Comment by jreidinger** [comment]
@aplanas btw it would be nice feature of github to see if user can merge PR or not itself, as yast prefer to have PRs merged by author especially if there are dependencies ( like in this case with patterns, packages, etc ).

**Comment by aplanas** [comment]
Thanks! But seems that I make a mistake, as I did not edit the spec file (to update the version) nor edited the changes file. I will prepare a new PR to fix this. I realized this after checking the package in OBS. Sorry for that.

**Comment by aplanas** [comment]
Missing package information update: https://github.com/yast/skelcd-control-MicroOS/pull/40

**Pull request merged**